### PR TITLE
update the header

### LINKED ISSUE #52: Replicate ubuntu.com nav style

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* removed search
* updated the html to latest vanilla

## QA

* download and `run serve`
* open any [page](http://0.0.0.0:8011/)
* see that the header looks as it should based on [vanilla](https://docs.vanillaframework.io/en/patterns/navigation) in all browser sizes, but is orange

## Screenshots

image

## Issues

Fixes #52

### CODE REVIEW

File: _sass/main.scss [hunk 1]
```diff
@@ -41,7 +41,16 @@ $color-link:          #E95420;
...
   &__logo {
-    margin: .5rem;
+    margin: $sp-x-small;
+    float: left;
+  }
+
+  &__link {
+    font-size: 1.25rem;
+
+    & > img {
```
Comment (grahambancroft) [inline _sass/main.scss]: Should we BEM this?

File: _includes/header.html [hunk 2]
```diff
@@ -1,38 +1,25 @@
...
-    <a class="p-navigation__link" href="/">
-      <img width="118" height="27" src="{{ site.assets_path }}2f6198a2-ubuntu-logo.png" alt="Ubuntu logo for print" />
-      <span>design</span>
-    </a>
+<header id="navigation" class="p-navigation">
+  <div class="row">
+    <div class="p-navigation__banner">
+      <div class="p-navigation__logo">
+        <a class="p-navigation__link" href="/">
+          <img width="118" height="27" src="{{ site.assets_path }}2f6198a2-ubuntu-logo.png" alt="Ubuntu logo for print" />
```
Comment (grahambancroft) [inline _includes/header.html]: Should this be an inline SVG as per ubuntu.com?

File: _sass/main.scss (hunk 1 above)
Comment (pmahnke) [reply _sass/main.scss]: probably, apparently, this pattern is being updated as such, so I will copy

File: _includes/header.html (hunk 2 above)
Comment (pmahnke) [reply _includes/header.html]: @grahambancroft , @anthonydillon has suggested an svg with the 'design' added, I have asked @spencerbygraves to create.

**Comment by pmahnke** [comment]
@grahambancroft ok, the image and text span have been replaced with an svg per you and @anthonydillon's comments... ready for re-review

File: _sass/main.scss (hunk 1 above)
Comment (grahambancroft) [reply _sass/main.scss]: Do you need the & > img { now?

**Pull request merged**